pkgs/development/ruby-modules/bundler-env/default.nix

```diff
@@ -6,7 +6,13 @@
 , tree
 }@defs:
 
-{ name, gemset, gemfile, lockfile, ruby ? defs.ruby, gemConfig ? defaultGemConfig
+{ gemset, gemfile, lockfile
+# the name is infered from the pName and the associated gem version
+, name ? null
+# if the pName is set, only the associated gem will have it's bins exposed
+, pName ? null
```

Comment:
The [Haskell generic builder](https://github.com/NixOS/nixpkgs/blob/97fd9058235d9067b8760efec07643d06eada514/pkgs/development/haskell-modules/generic-builder.nix#L5) calls this `pname` with a lowercase "n"; it might be worthwhile to do the same here for the sake of consistency.